.github/workflows/scorecard.yml

```diff
@@ -0,0 +1,59 @@
+# This workflow creates an OpenSSF scorecard, see https://scorecard.dev/
+
+name: Scorecard supply-chain security
+on:
+  # For Branch-Protection check. Only the default branch is supported. See
+  # https://github.com/ossf/scorecard/blob/main/docs/checks.md#branch-protection
+  branch_protection_rule:
+  # To guarantee Maintained check is occasionally updated. See
+  # https://github.com/ossf/scorecard/blob/main/docs/checks.md#maintained
+  schedule:
+    - cron: '40 23 * * 5'
+  push:
+    branches: [ "main" ]
```

Comment:
Why not executing the scans for PRs as well?

Reply:
The default workflow as configured going through https://github.com/marketplace/actions/ossf-scorecard-action is not set up to scan PRs. Considering the [checks that are executed](https://scorecard.dev/#the-checks) I think it doesn't make sense as they cannot all be influenced just by a PR.